### PR TITLE
New Cisco and Validated Patterns team partnership blog

### PR DESCRIPTION
**Pull request by claudiol**
* Deploying Multicloud GitOps with Cisco FlashStack Data Center and Portworx Enterprise
Reviewed-by: Paniraja Koppa pkoppa@cisco.com
Reviewed-by: Andrew Beekhof abeekhof@redhat.com
- CISCO Blog for MCG FlashStack and Portworx
- Added header for blog
- Removed alias from header

### CODE REVIEW

**Comment by mbaldessari** [comment]
This is an automated message:

You can preview this docs PR at http://398.docs-pr.validatedpatterns.io
Note that they get generated every five minutes, so please wait a bit.

File: content/blog/2024-01-16-deploying-mcg-with-cisco-flashstack-portworx.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+The deployment of [Red Hat OpenShift Container Platform](https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform) serves as a cornerstone of this solution. The Multicloud GitOps Validated Pattern seamlessly deploys additional solution components, such as [Red Hat Advanced Cluster Management](https://www.redhat.com/en/technologies/management/advanced-cluster-management), [Red Hat Advanced Cluster Security](https://www.redhat.com/en/technologies/cloud-computing/openshift/advanced-cluster-security-kubernetes), [Red Hat Quay](https://www.redhat.com/en/technologies/cloud-computing/quay), and[Portworx Enterprise](https://portworx.com/services/kubernetes-storage/) providing a complete platform. Leveraging the robustness of Cisco UCS server platforms, organizations can manage and orchestrate containerized applications on-premises and across multi cloud environments. The versatility of Red Hat OpenShift, coupled with the reliability of Cisco UCS servers, forms a powerful synergy driving efficiency and performance.
```
Comment (mhjacks) [inline content/blog/2024-01-16-deploying-mcg-with-cisco-flashstack-portworx.md:20]: MCGO doesn't deploy Quay, normally. Did Cisco include that?

```and[Portworx Enterprise]```
^ Need a space here

**Review by mhjacks — CHANGES_REQUESTED** [review]
Minor nit on spacing. I'm not sure we should mention Quay - I looked at the whitepaper and didn't see a reference there either.

**Comment by mhjacks** [comment]
Thanks for the update.

/lgtm

**Pull request merged**